Pokemon Repository

slabserif opened the pull request:
Added a pokemon repository and a new interface to parse the excel file.
Ethans a little bitch.

slabserif committed:
Pokemon Repository

Added a pokemon repository and a new interface to parse the excel file.
Ethans a little bitch.

Gekctek commented on src/Core/PokemonExcelRepository.cs:
```diff
@@ -0,0 +1,64 @@
+    {
+        public static PokemonExcelRepository Create()
+        {
+            //parse file
```
Put a TODO statement or finish it

slabserif committed:
Parsing pokedex

Adding parsing of pokedex file and converting them into pokemon objects
using reflection (Ethan)
